### PR TITLE
refactor(site): add presentation draft

### PR DESCRIPTION
**Pull request by ala-n**

### CODE REVIEW

File: site/package.json [hunk 1]
```diff
@@ -17,14 +17,15 @@
     "watch:ts": "webpack --watch",
     "watch:less": "chokidar --initial \"../node_modules/@exadel/esl/modules/**/*.less\" \"src/**/*.less\" -c \"npm run build:less\"",
     "watch:eleventy": "eleventy --serve --incremental -- --env=development",
-    "minify:css": "cleancss -o \"dist/bundles/localdev.css\" \"dist/bundles/localdev.css\"",
+    "minify:css": "cleancss -o \"dist/bundles/all.css\" \"dist/bundles/all.css\"",
```
Comment (ala-n) [inline site/package.json:20]:
```suggestion
    "minify:css": "cleancss -o \"dist/bundles/site.css\" \"dist/bundles/site.css\"",
```

File: site/views/overview/index.njk [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+---
+
+<div class="d-flex container" style="height: 100dvh; ">
+  <div class="m-auto markdown-container">
+    <h1> ESL Presentations </h1>
+    <ul>
+      <li>
+        <a class="btn large btn-link" href="{{ '/overview/quick' | url }}"> Quick Overview </a>
+      </li>
+{#      <li>#}
```
Comment (NastaLeo) [inline site/views/overview/index.njk:14]: Sorry, I missed the difference between quick and full overview

**Review by NastaLeo — APPROVED** [review]

File: site/views/overview/index.njk (hunk 2 above)
Comment (ala-n) [reply site/views/overview/index.njk:14]: There is no full presentation for now (it's going to be detailed dev presentation)
The quick one is the one that we already had used during solutions presentation

File: site/src/presentation/layout.less [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  background-size: 1000px 500px;
+  animation: p-slow-move-bg 90s infinite linear;
+
+  &::before {
+    content: '';
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
```
Comment (dshovchko) [inline site/src/presentation/layout.less:45]:
```suggestion
    inset: 0;
```

File: site/src/presentation/slide.less [hunk 4]
```diff
@@ -0,0 +1,39 @@
+.presentation-slide {
+  display: flex;
+  position: relative;
+  max-height: 100%;
+
+  @media print {
+    // A4 landscape height
+    // min-height: 210mm;
```
Comment (abarmina) [inline site/src/presentation/slide.less:8]: cosmetic: do we need this as a comment?
```suggestion
    // min-height: 210mm;
```

File: site/views/overview/quick/02-core/03-tag-vs-attribute.njk [hunk 5]
```diff
@@ -0,0 +1,64 @@
+---
+title: ESLBaseElement vs ESLMixinElement
+hash: tag-vs-attribute
+tags: -overview-quick
```
Comment (abarmina) [inline site/views/overview/quick/02-core/03-tag-vs-attribute.njk:4]:
```suggestion
tags: overview-quick
```

**Review by abarmina — APPROVED** [review]

File: site/views/overview/quick/02-core/03-tag-vs-attribute.njk (hunk 5 above)
Comment (ala-n) [reply site/views/overview/quick/02-core/03-tag-vs-attribute.njk:4]: That was in purpose, that's the easiest way to exclude slide

**Review by dshovchko — APPROVED** [review]

**Pull request merged**